Tasks/Gradle/CodeAnalysis/checkstyle.gradle

```diff
@@ -16,8 +16,12 @@ rootProject {
 
                 proj.apply plugin: 'checkstyle'
                 checkstyle {
-                    println(rootDir);
-                    configFile rootProject.file('checkstyle.xml')
+                    // Groovy way of resolving the current directory  
```

Comment:
By "Groovy", I assume you're not referring to the Groovy language that runs on a JVM?
"Current directory" = "directory containing this file"?